### PR TITLE
Configuration to stream Docker Events logs to Stackdriver Logging.

### PR DESCRIPTION
**Pull request by nataliasakowska**

### CODE REVIEW

File: scripts/konlet-startup [hunk 1]
```diff
@@ -28,6 +28,28 @@ if [[ $? -ne 0 ]]; then
   ACTUAL_KONLET_VERSION="${KONLET_VERSION_TAG}"
 fi
 
+sudo sh -c 'echo "[Unit]
```
Comment (vriesk) [inline scripts/konlet-startup]: The startup script should not need sudo.

File: scripts/konlet-startup [hunk 2]
```diff
@@ -28,6 +28,28 @@ if [[ $? -ne 0 ]]; then
...
+<source>
+  @type systemd
+  filters [{ \"_SYSTEMD_UNIT\": \"stackdriver-docker-events-collector.service\" }]
+  pos_file /var/log/google-fluentd/stackdriver-docker-events.log.pos
+  read_from_head true
+  tag cos_docker_events
+</source>" >> /etc/stackdriver/logging.config.d/fluentd-lakitu.conf'
+
+sudo systemctl start stackdriver-docker-events-collector.service
+sudo systemctl restart stackdriver-logging.service
```
Comment (vriesk) [inline scripts/konlet-startup]: Are you positive that systemctl ran from inside of a systemctl-launched script won't cause any deadlock issues?

File: scripts/konlet-startup [hunk 3]
```diff
@@ -28,6 +28,28 @@ if [[ $? -ne 0 ]]; then
   ACTUAL_KONLET_VERSION="${KONLET_VERSION_TAG}"
 fi
 
+sudo sh -c 'echo "[Unit]
+Description=A daemon to collect Docker events and send to to journal logs.
```
Comment (vriesk) [inline scripts/konlet-startup]: I'd very much rather see this just included in COS image, but disabled by default.

Comment (xueweiz) [reply scripts/konlet-startup]: We prefer to have a more robust solution than a systemd script. i.e. We need the solution to handle Docker daemon restart properly. For context: b/79548044#comment9

**Pull request closed without merging**